skrub/_reporting/_plotting.py

```diff
@@ -115,6 +117,67 @@ def _adjust_fig_size(fig, ax, target_w, target_h):
     fig.set_size_inches((w, h))
 
 
+def _get_range(values, frac=0.2, factor=3.0):
+    min_value, low_p, high_p, max_value = np.percentile(
+        values, [0, frac * 100.0, (1.0 - frac) * 100.0, 100.0]
+    )
+    delta = high_p - low_p
+    if not delta:
+        return min_value, max_value
+    margin = factor * delta
```

Comment:
Interesting heuristic, how did you get these default parameters?

Reply:
from the random number generator in my head :sweat_smile:  as mentioned I think the heuristic can definitely be refined or replaced with somehting else

Reply:
I thought of using the inter-quantile range because that's what box plots do, but I think we could read more about simple outlier detection methods